client/components/confetti-animation/index.tsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect } from 'react';
+import confetti from 'canvas-confetti';
+
+const defaultColors = [ '#889BF2', '#C3CDF9', '#6079ED' ];
+
+const fireConfetti = ( colors: string[] ) => {
```

Comment:
+1 looks really great! I realize there wasn't a design animation provided so if it's simple to do could we start the confetti with a wider spread and a little higher up on the screen? I'm thinking that the current starting point is too tight and the "explosion" should be a bit more subtle.

Reply:
In the designs I was envisioning the animation would start at what ~1sec in the video looks like. 
<img width="395" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/5121465/3f9e39aa-7e42-423f-8662-f68428ca0158">